docs/source/about/changelog.rst

```diff
@@ -28,6 +28,13 @@ Unreleased
 - :pull:`1118` - `module_from_template` is broken with a recent release of `requests`
 - :pull:`1131` - `module_from_template` did not work when using Flask backend
 
+**Added**
+
+- :pull:`1165` - Allow concurrent renders of distinct components - enable this
```

Comment:
Might want to add a note that this does **not** allow one component to concurrently render itself multiple times.

I'm sure someone will

Reply:
Also we need to decide if `distinct` or `discrete` is the proper word to describe exclusive component trees.

We just need to keep our verbiage consistent. I've been using the word `discrete`, not to say that is correct.

Reply:
It probably also makes sense to be more explicit here and say "(distinct | discrete) component trees" rather than just "distinct components".